packages/docs/src/components/docsearch/doc-search.tsx

```diff
@@ -66,43 +66,43 @@ export const DocSearch = component$((props: DocSearchProps) => {
 
   const searchButtonRef = useRef();
 
-  return (
-    <div
-      class="docsearch"
-      window:onKeyDown$={(event) => {
-        function open() {
-          // We check that no other DocSearch modal is showing before opening
-          // another one.
-          if (!document.body.classList.contains('DocSearch--active')) {
-            state.isOpen = true;
-          }
-        }
-        if (
-          (event.key === 'Escape' && state.isOpen) ||
-          // The `Cmd+K` shortcut both opens and closes the modal.
-          (event.key === 'k' && (event.metaKey || event.ctrlKey)) ||
-          // The `/` shortcut opens but doesn't close the modal because it's
-          // a character.
-          (!isEditingContent(event) && event.key === '/' && !state.isOpen)
-        ) {
-          // FIXME: not able to prevent
-          // event.preventDefault();
+  useBrowserVisibleTask$(() => {
+    window.addEventListener('keydown', (event) => {
+      event.preventDefault();
```

Comment:
with this in place, you're not able to type anything into the search form since all key strokes are prevented atm 😄

Reply:
I have fixed that :)